### PR TITLE
Feat: add top level locator functions

### PR DESCRIPTION
**Pull request by Maksimka101**

### CODE REVIEW

**Comment by f-person** [comment]
Thanks for the PR! Could you also add tests to keep the coverage at 100%?

**Comment by Maksimka101** [comment]
💯😎

**Comment by Maksimka101** [comment]
Soo, will you merge and publish it?)

File: README.md [hunk 1]
```diff
@@ -220,6 +220,31 @@ void logout() {
...
+// which basically will call this method on the `Toor.instance`.
+final httpClientSingleton = registerLazySingleton<IHttpClient>(
+  DioHttpClientImpl.new,
+);
+```
+
+Toor provides a top level functions to register locators 
+in the global instance of the toor. So instead of creating
+a top level `toor` variable which holds `Toor.instance`
+and then using its methods you can
```
Comment (Maksimka101) [inline README.md]: Seems like I forgot to remove the old text

**Pull request merged**